frontend/tickets-online/src/user/actions/Actions.js

```diff
@@ -23,24 +24,29 @@ function logInUser(email, password) {
 
     return fetch(url + 'api/account/login', requestOptions)
       .then(function(response) {
-        if (response.status === 400) {
+        if (response.status !== 200) {
```

Comment:
I recommend to use one of HttpStatusCode packages from npm when you need compare status code. It will look like response.status !== HttpStatusCode.OK what is more readable